Client/Frontend/Browser/Playlist/VideoPlayer/DataURIParser.swift

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+class DataURIParser {
```

Comment:
Where are you using this?

Reply:
Currently unused, but in the future it will be used because some videos come back as base-64 encoded.